docs/sphinx/api/public_api.rst

```diff
@@ -92,10 +92,10 @@ Functions
 - :cpp:func:`hpx::parallel::v1::swap_ranges`
 - :cpp:func:`hpx::parallel::v1::unique`
 - :cpp:func:`hpx::parallel::v1::unique_copy`
-- :cpp:func:`hpx::parallel::v2::for_loop`
-- :cpp:func:`hpx::parallel::v2::for_loop_strided`
-- :cpp:func:`hpx::parallel::v2::for_loop_n`
-- :cpp:func:`hpx::parallel::v2::for_loop_n_strided`
+- :cpp:func:`hpx::for_loop`
+- :cpp:func:`hpx::for_loop_strided`
+- :cpp:func:`hpx::for_loop_n`
+- :cpp:func:`hpx::for_loop_n_strided`
```

Comment:
Should these go into `hpx::experimental` instead (it's not in the standard yet, after all)

Reply:
Unsure. I'd be inclined to leave it here because they're stable (enough) in HPX, even though they are experimental in the standard. Our executors are also not in the standard, but not in the experimental namespace.